torchsynth/config.py

```diff
@@ -7,7 +7,8 @@
 # and abstract synth parameter randomization
 # (https://github.com/torchsynth/torchsynth/issues/253)
 # are non-reproducible unless batch_size == BATCH_SIZE_FOR_REPRODUCIBILITY.
-BATCH_SIZE_FOR_REPRODUCIBILITY = 128
```

Comment:
Docs need to be updated to explain what each of these mean and the comment above

Reply:
Created an issue for it: #331